README.md

```diff
@@ -39,6 +39,7 @@ shows which versions of each generator are inside the docker image.
 
 | Docker image           | uniffi-rs version     | uniffi-bindgen-cs version | uniffi-bindgen-go version | uniffi-bindgen-cpp version |
 |------------------------|-----------------------|---------------------------|---------------------------|----------------------------|
+| v0.25.0-3              | v0.3.0+0.25.0         | v0.8.0+v0.25.0            | v0.2.0+v0.25.0            | **v0.4.0+v0.25.0**         |
```

Comment:
It would be nice to bolden `uniffi-bindgen-cs` version too, its easier to see which version were changed.
```suggestion
| v0.25.0-3              | v0.3.0+0.25.0         | **v0.8.0+v0.25.0**        | v0.2.0+v0.25.0            | **v0.4.0+v0.25.0**         |
```

Reply:
Oh, now it makes sense what these were for :D